src/main/resources/liquibase/harness/change/changelogs/mysql/createProcedure.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+        <!--https://docs.liquibase.com/change-types/community/create-procedure.html-->
+        <!--Liquibase community doesn't support snapshot generation for procedures thus
+            expectedSnapshot file will be empty for this change type-->
+    <changeSet author="as" id="1">
+        <createProcedure
+                comments="test procedure"
+                dbms="mysql"
+                encoding="UTF-8"
+                path="createProcedure.sql"
+                procedureName="test_procedure"
+                procedureText="CREATE PROCEDURE test_procedure()
+
+                               BEGIN
+
+                               END;"
+                relativeToChangelogFile="true">
+        </createProcedure>
+        <rollback/>
```

Comment:
In such implementation dropProcedure test case depends on createProcedure testcase. We don't guarantee execution order. In general tests shouldn't depend on each other. Are there any restrictions why we can'd drop procedure in rollback section here and create procedure + immediately drop procedure in dropProcedure test case?

Reply:
Fixed